main.nf

```diff
@@ -888,7 +895,9 @@ process featureCounts {
     """
     featureCounts -a $gtf -g gene_id -o ${bam_featurecounts.baseName}_gene.featureCounts.txt -p -s $featureCounts_direction $bam_featurecounts
     featureCounts -a $gtf -g gene_biotype -o ${bam_featurecounts.baseName}_biotype.featureCounts.txt -p -s $featureCounts_direction $bam_featurecounts
-    cut -f 1,7 ${bam_featurecounts.baseName}_biotype.featureCounts.txt > ${bam_featurecounts.baseName}_biotype_counts.txt
+    cut -f 1,7 ${bam_featurecounts.baseName}_biotype.featureCounts.txt | tail -n 7 > ${bam_featurecounts.baseName}_biotype_counts.txt
+    cat $biotypes_header  ${bam_featurecounts.baseName}_biotype_counts.txt >> tmp_file
+    mv tmp_file ${bam_featurecounts.baseName}_biotype_counts.txt
```

Comment:
Instead of making a temp file and then renaming it, just make the first generated file called something else (`${bam_featurecounts.baseName}_biotype_counts_raw.txt`) then cat directly to the desired final filename.